doc/nrf/security/tfm/tfm_supported_services.rst

```diff
@@ -160,10 +160,55 @@ The following table lists the isolation level support in Nordic Semiconductor's
      - Not supported
      - Maximum firmware isolation, with each Secure Partition sandboxed and only permitted to access its own resources.
 
+.. _ug_tfm_ncs_zephyr_differences:
+
+Differences between TF-M in the |NCS| and upstream Zephyr
+*********************************************************
+
+The TF-M implementation in the |NCS| differs from the implementation in :term:`upstream <Upstream repository>` Zephyr.
+These are the main differences:
+
+* Board support:
+
+  * The :ref:`ug_tfm_supported_services_tfm_services` implemented in the |NCS| are not recommended for use with boards in upstream Zephyr because these boards have limited TF-M support:
+
+    * Upstream Zephyr only fully supports TF-M for nRF91 Series DKs and nRF5340 DK.
+    * Upstream Zephyr's TF-M integration for nRF54L15 is limited to the :zephyr:code-sample:`tfm_ipc` sample.
+    * In upstream Zephyr, TF-M integration for boards based on nRF5340 (other than the nRF5340 DK) is not supported.
+
+* TF-M configuration:
+
+  * The |NCS| allows for more customization of TF-M features:
+
+    * The |NCS| provides a :ref:`minimal build <tfm_minimal_build>` option that is enabled by default for nRF53 and nRF91 Series devices.
+    * The |NCS| offers a :ref:`configurable build <tfm_configurable_build>` that allows for more fine-grained control over TF-M features.
+
+  * Upstream Zephyr uses predefined, basic TF-M profiles that might result in larger memory footprint.
+
+* Logging:
+
+  * The |NCS| provides enhanced :ref:`logging capabilities <ug_tfm_logging>` with options to configure UART instances.
+  * The |NCS| allows TF-M and the application to share the same UART for logging.
+
+* Security features:
+
+  * The |NCS| integrates TF-M with :ref:`nrf_security` to enable hardware acceleration using Arm CryptoCell.
```

Comment:
The CryptoCell is only relevant for the 53 and 91 devices. The 54L doesn't have it, it uses another crypto acceleration. I think you can generalize it by removing the Arm CryptoCell part and just say that it integrates nrf_security which enables crypto hardware acceleration.